hathor/manager.py

```diff
@@ -573,8 +571,6 @@ def _initialize_components_new(self) -> None:
         self.tx_storage.pre_init()
         assert self.tx_storage.indexes is not None
 
-        self._bit_signaling_service.start()
```

Comment:
I don't remember the specific reason anymore, but it was something related to initialization order. I reverted it locally now but couldn't reproduce the problem. It may be something that breaks if a future PR if this change is not made. Since I don't think it's too relevant, I would rather leave it like this, but I can change it back if you prefer.

Reply:
I guess this change might have some effect. If the tests keep passing, I'd say let's remove this change. We can do it later if needed anyways.

Reply:
Ok, reverted in https://github.com/HathorNetwork/hathor-core/pull/1014/commits/4a886180770b3e8549573cffc6ce8ad6df4c4fa7

Reply:
Thanks!